client/src/modules/profiles/pages/userProfile.tsx

```diff
@@ -56,32 +56,37 @@ export const UserProfilePage = () => {
 
   return (
     <div>
-      {user ? (
+      {data ? (
         <>
           <Heading as="h1" marginBlock={'.5em'}>
             Profile
           </Heading>
           <Heading as="h2" size={'lg'}>
-            Welcome {getNameText(user.name)}
+            Welcome {getNameText(data.userInformation?.name)}
           </Heading>
-          {user.admined_chapters.length > 0 && (
+          {data.userInformation?.admined_chapters.length > 0 && (
             <>
               <Heading as="h2" marginBlock={'.5em'} size="md">
                 You are an administrator for these Chapters:
               </Heading>
               <Flex marginTop={'1em'} flexDirection={'column'} gap={4}>
-                {user.admined_chapters.map(({ name, id }) => (
+                {data.userInformation?.admined_chapters.map(({ name, id }) => (
                   <Link key={id} href={`/chapters/${id}`}>
                     {name}
                   </Link>
                 ))}
               </Flex>
             </>
           )}
-
+          <Heading as="h2" marginTop={'2em'} size="lg" fontWeight={500}>
+            This will change the setting in{' '}
```

Comment:
```suggestion
            Email address:{' '}
```
This seems clearer.  Ideally the styling should be uniform across Email address, Name and Profile picture, but that doesn't have to be this PR.